apps/projects/app/components/Panel/NewBountyAllocation/NewBountyAllocation.js

```diff
@@ -246,6 +246,12 @@ class NewBountyAllocation extends React.Component {
   }
 }
 
+const HoursInput = styled(TextInput.Number)`
```

Comment:
We could use [attrs](https://www.styled-components.com/docs/api#attrs) here to pass some initial attributes that should help to validate the field, like min, max or step:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#Additional_attributes
Then maybe we could save some conditional code at lines 91-94

Reply:
Great tip - added :)